assets/constants.ts

```diff
@@ -7095,5 +7095,6 @@ export default {
   "CONTRIBUTOR_CERTIFICATE_WIDTH": 1493,
   "CONTRIBUTOR_CERTIFICATE_HEIGHT": 1313,
   "BRANCH_NAME": "",
-  "SHORT_COMMIT_HASH": ""
+  "SHORT_COMMIT_HASH": "",
+  "CSRF_SECRET_INSTANCE_ID": "csrf_secret"
```

Comment:
Why is this here? If only the backend fetches it, keep it as a constant in the backend (the constants.ts file is only for stuff that needs to be referenced in both the frontend and backend).

Reply:
Got it, removed.